### PR TITLE
Bring polaroids closer

### PR DESCRIPTION
**Pull request by mei-lu**
image

### CODE REVIEW

**Comment by julesparra** [comment]
looks like u need to pull, there are just a couple commits i made yesterday that aren't on this branch

File: scss/main.scss [hunk 1]
```diff
@@ -921,21 +911,16 @@ a:hover {
...
   }
 
-  /*@media only screen and (max-width: 950px) {
-    width: 15rem !important;
-
-    &.polaroid1 {
-      margin-left: 15rem !important;
-    }
-
+  @media only screen and (max-width: 950px) {
```
Comment (julesparra) [inline scss/main.scss:919]: can you center the polaroids on this tablet/narrow desktop view (needed when width is 960px - 1180px)
Screen Shot 2021-08-19 at 9 51 29 AM

**Review by julesparra — CHANGES_REQUESTED** [review]
looks sm better on mobile, just a nitpick on tablet🐸 ☁️

**Comment by mei-lu** [comment]
> looks like u need to pull, there are just a couple commits i made yesterday that aren't on this branch

image

Eh? I am on the right branch and I pulled before. Looks like there's nothing to pull?

**Comment by mei-lu** [comment]
image
tada ~

**Review by julesparra — APPROVED** [review]

**Pull request merged**